### ISSUE TITLE
comment HTML syntax with ctrl + / in PHP File

### ISSUE THREAD

**Issue by baadillahnabil** [Microsoft/vscode#10797]
- VSCode Version: 1.4.0
- OS Version: Windows 10

when i press ctrl+/ in html syntax, it show comment symbol like this // not this 
what i'm saying is when i press ctrl+/ in html syntax it should write this  because it's the proper comment for html syntax
Thanks, and i apologize for my bad english :)

**Comment by rmunn** [comment]
I just tried Ctrl+/ in an HTML file, and I got proper `<!-- -->` comments. (VS Code 1.4.0 on Linux). Which language mode is your editor in when you edit the HTML file? (It shows up on the far right-hand edge of the status bar, just next to the "smiley face" icon). If it doesn't say HTML, try clicking on it and selecting a different language mode -- maybe you've gotten `.html` files configured to open in a different language mode, like PHP or something.
